### ISSUE TITLE
wrong execution result

### ISSUE THREAD

**Issue by yjiawqgj** [herlon214/iptv-recording#1]
duration Error parsing cron: end of range (42) above maximum (23): 42

**Comment by herlon214** [comment]
@yjiawqgj thanks for reporting. Could you give more context?
I think you're using the cron wrong, you got switched hour and minute probably. Take a look at https://crontab.guru/ and check your cron.

**Comment by yjiawqgj** [comment]
items:
  - name: Live Name
    url: http://10.10.10.1:10000/udp/239.253.228.94:8000#This is a multicast-to-unicast address, which can be played locally.
    fileName: recording-$date # $date will be transformed to "2022-01-11.1720" (for example)
    folder: /mnt/data
    schedule: "0 43 * * *" # Cron style
    duration: 1m20s
This is to copy your recording.yaml file, just change 04 to 43, but follow the prompt to change the number within 23 without prompting an error. But if you change it to a number within 23, iptv-rec just executes to the second step. The time has come but no recording has been made.
2022/08/01 09:01:00 Found 1 items to record
2022/08/01 09:01:00 ---------------------
I executed "iptv-rec.exe local" under windows

**Comment by herlon214** [comment]
thanks for the info @yjiawqgj.
As I was saying before, you switched hour and minutes: https://crontab.guru/#0_43_*_*_*
![image](https://user-images.githubusercontent.com/3419441/182573691-c0f2c838-cacf-43da-a0fa-c831e4c23113.png)

Could you try using: `*/5 * * * *` so it will start recording every **5 minutes** for **1m20s**?

**Comment by yjiawqgj** [comment]
Yes, it is the cron usage error, because the tutorials I saw on the Internet put the cron format. The first one written in the tutorial is the second. I wrote it according to his instructions, so it will keep making mistakes. Now there is a new problem that Failed to stop recording: TerminateProcess: Access is denied. This prompt, running CDM with administrator is still wrong.
